topology/probes/istio/virtualservice.go

```diff
@@ -48,3 +50,120 @@ func (h *virtualServiceHandler) Dump(obj interface{}) string {
 func newVirtualServiceProbe(client *kiali.IstioClient, g *graph.Graph) k8s.Subprobe {
 	return k8s.NewResourceCache(client.GetIstioNetworkingApi(), &kiali.VirtualService{}, "virtualservices", g, &virtualServiceHandler{})
 }
+
+type virtualServicePodLinker struct {
+	graph    *graph.Graph
+	vsCache  *k8s.ResourceCache
+	podCache *k8s.ResourceCache
+}
+
+func getHostsByVirtualService(vs *kiali.VirtualService) []string {
+	hosts := []string{}
+	if vs.Spec["http"] == nil {
+		return hosts
+	}
+	https := vs.Spec["http"].([]interface{})
```

Comment:
`https` is misleading rather use `httpList`

Reply:
DONE